### PR TITLE
fix(status): position and size of status

### PR DESCRIPTION
**Pull request by AlexZakablukov**
**What this PR does** 📖
fix position and size of status (on the message list on the sidebar it was slightly wrong)

**Which issue(s) this PR fixes** 🔨
[AP-1462](https://satellite-im.atlassian.net/browse/AP-1462)

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

File: components/ui/User/State/State.less [hunk 1]
```diff
@@ -4,36 +4,40 @@
...
-    mask: radial-gradient(circle 8px at 85.35% 85.35%, #0000 98%, #000) 100%
-      100%/100% 100% no-repeat;
+    mask: radial-gradient(circle 8px at 86% 86%, #0000 98%, #000) 100% 100%/100%
+      100% no-repeat;
     overflow: hidden;
+    &.is-large {
+      mask: radial-gradient(circle 12px at 86% 86%, #0000 98%, #000) 100% 100%/100%
+        100% no-repeat;
+    }
   }
```
Comment (josephmcg) [inline components/ui/User/State/State.less:14]: I would probably setup a variable, then change the value of the variable for .is-large. something like this
`  .circle {
    --circle-size: 8px;
    mask: radial-gradient(circle var(--circle-size) at 86% 86%, #0000 98%, #000)
      100% 100%/100% 100% no-repeat;
    overflow: hidden;
    &.is-large {
      --circle-size: 12px;
    }
  }`

File: components/views/chat/group/Group.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
     <UiCircle
       :type="src ? 'image' :'random'"
       :seed="address"
-      :size="35"
+      :size="36"
```
Comment (josephmcg) [inline components/views/chat/group/Group.html:6]: any place you're passing in the default of 36 can be removed, right?

**Review by josephmcg — CHANGES_REQUESTED** [review]

File: components/ui/User/State/State.less (hunk 1 above)
Comment (AlexZakablukov) [reply components/ui/User/State/State.less:14]: great point, thx)

File: components/views/chat/group/Group.html (hunk 2 above)
Comment (AlexZakablukov) [reply components/views/chat/group/Group.html]: ofc, I removed them

**Comment by AlexZakablukov** [comment]
fixed

**Review by josephmcg — APPROVED** [review]
great, thanks Alex

**Review by stavares843 — APPROVED** [review]

**Comment by AlexZakablukov** [comment]
fixed

**Review by phillsatellite — APPROVED** [review]

**Comment by molimauro** [comment]
On non-retina screens i see the problem persisting. I'm fixing it, in the meanwhile let this PR go in since it contains useful minor improvements :)

**Review by molimauro — APPROVED** [review]

**Review by stavares843 — APPROVED** [review]

**Pull request merged**